### PR TITLE
Include the esp_vfs_fat.h header

### PR DESCRIPTION
**Pull request by derchr**
I needed to use the FatFs driver and want to share my modifications.

### CODE REVIEW

File: src/include/esp-idf/bindings.h [hunk 1]
```diff
@@ -46,6 +46,10 @@
 #include "esp_tls.h"
 #endif
 
+#ifdef ESP_IDF_COMP_ESP_VFS_FAT_ENABLED
+#include "esp_vfs_fat.h"
+#endif
+
 #ifdef ESP_IDF_COMP_APP_UPDATE_ENABLED
```
Comment (ivmarkov) [inline src/include/esp-idf/bindings.h:53]: I don't think a component named "esp_vfs_fat" exists? Usually, the components are named after their corresponding names in the `components/` folder of ESP-IDF (modulo replacing - with _). I don't see `esp_vfs_fat` there. What _is_ there, is `fatfs`.

**Review by ivmarkov — CHANGES_REQUESTED** [review]

**Comment by ivmarkov** [comment]
I'll merge and then fix myself.

**Comment by ivmarkov** [comment]
Accommodated here: https://github.com/esp-rs/esp-idf-sys/commit/1d9adac0aa916dcdfda64fd48084aa3ed024e6bc

**Comment by derchr** [comment]
Thanks for adding this.
What I noticed is that somehow the first 3 bytes of all files that are being read from the filesystem get truncated. I'm pretty sure this wasn't happening last week as I've successfully loaded some images from the fs.
Also, using my version of the pull request does not resolve this either, so the changes in esp-idf-sys are most likely not the problem here.

This all seems very weird. Is there maybe another moving part that could have caused this?

**Comment by ivmarkov** [comment]
Without a simple repro case it is difficult to say what is going on

**Pull request closed without merging**